client/components/data/query-top-posts/index.jsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { isUndefined } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { isRequestingTopPosts } from 'state/stats/top-posts/selectors';
+import { requestTopPosts } from 'state/stats/top-posts/actions';
+
+class QueryTopPosts extends Component {
+	static defaultProps = {
+		requestTopPosts: () => {},
+		period: 'day',
+		num: 1,
+	};
+
+	static propTypes = {
+		siteId: PropTypes.number,
+		date: PropTypes.string,
+		period: PropTypes.string,
+		num: PropTypes.number,
+		requestingTopPosts: PropTypes.bool,
+		requestTopPosts: PropTypes.func,
+	};
```

Comment:
Is there a way to pass in the max number of results to return? I don't see this referenced anywhere in the docs or `QueryTopPosts` component. As a user of this component, it would be unclear to me how many posts I should expect to be returned.